server/config.ts

```diff
@@ -15,30 +15,35 @@ const nonSensitiveSharedConfig = {
 }
 
 const keysToPullFromEnv = [
-  'SophtronClientId',
-  'SophtronClientSecret',
-
-  'UcpAuthClientId',
-  'UcpAuthClientSecret',
-  'UcpAuthEncryptionKey',
-
-  'CryptoKey',
-  'CryptoIv',
-
-  'Demo',
-  'DefaultProvider',
-  'Port',
-
-  'HostUrl',
-  'WebhookHostUrl',
-  'LogLevel',
-
-  'Env',
-
-  'RedisServer',
-  'RedisCacheTimeSeconds',
-  'ResourcePrefix',
-  'ResourceVersion'
+  "CryptoKey",
+  "CryptoIv",
+
+  "AuthProvider", 
+
+  "LocalAuthEncryptionKey", 
+  "LocalAuthPhrase",
+  
+  "SophtronClientId",
+  "SophtronClientSecret",
+
+  "UcpAuthClientId",
+  "UcpAuthClientSecret",
+  "UcpAuthEncryptionKey",
+
+  "Demo",
+  "DefaultProvider",
+  "Port",
+
+  "HostUrl",
+  "WebhookHostUrl",
+  "LogLevel",
+  
+  "Env",
+
+  "RedisServer",
+  "RedisCacheTimeSeconds",
+  "ResourcePrefix",
+  "ResourceVersion",
```

Comment:
We're using `standard-with-typescript` styles which prefers single quotes over double quotes. Please install a linter in your IDE to make these style requirements more obvious as you're coding. Eventually, when we get all the typescript errors resolved there will be a pipeline to check for linting errors and block pull requests. But we're trying to follow common patterns to make this repo more uniform and easy to read for future devs who will be reading or contributing to the code.